beacon_node/store/src/lib.rs

```diff
@@ -180,6 +183,11 @@ pub trait Store<E: EthSpec>: Sync + Send + Sized + 'static {
     }
 }
 
+pub enum StoreOp {
```

Comment:
Can this enum get a one-line doc comment, and a `TODO` to add `InsertBlock`, `InsertState`, with a link to #692?

Reply:
If the need arises, `InsertBlock` and `InsertState` will be added in its own time.  Does lack of TODO somehow makes that harder?  I also think this place should be linked from  #692, not the other way round.  After all, you start work by going to a GitHub issue, not with grepping TODOs.